src/symbols/Node.tsx

```diff
@@ -261,7 +267,8 @@ export const Node: FC<NodeProps> = ({
           active: combinedActiveState,
           opacity: selectionOpacity,
           animated,
-          node
+          node,
+          selected: isSelected
```

Comment:
>I don't see this used anywhere

Here

Reply:
You need to pull the selected from the state slice for perf reasons

Reply:
But isSelected uses everywhere in this component.

Reply:
Nevermind I was confused by the `selected` type addition